src/components/Picker.vue

```diff
@@ -239,10 +261,42 @@ export default {
     },
     onEmojiEnter(emoji) {
       this.previewEmoji = emoji
+      this.previewEmojiIdx = -1
+      this.previewEmojiCategoryIdx = -1
     },
     onEmojiLeave(emoji) {
       this.previewEmoji = null
     },
+    onArrowLeft() {
+      if (this.previewEmojiIdx > 0) {
+        this.previewEmojiIdx -= 1
+      } else {
+        this.previewEmojiCategoryIdx -= 1
+        if (this.previewEmojiCategoryIdx < 0) {
+          this.previewEmojiCategoryIdx = 0
+        } else {
+          this.previewEmojiIdx =
+            this.categories[this.previewEmojiCategoryIdx].emojis.length - 1
+        }
+      }
```

Comment:
Can this logic be simplified? Looks not really good.

The idea is that we track the active category and active emoji inside the category. When we go to the first emoji in the currently active category, we switch to the last emoji in the previous category.

We also need to make sure we stop at the first category.
A similar logic is used below for the right arrow.

Note: I first implemented this by using `allEmojis` mapping (idx -> emoji id), but we also need to know the active category because:

- Emojis can be in more that one category (the 'Frequently Used' category contains emojis from other categories). So with `allEmojis`  implementation the emoji was highlighted in all categories at the same time (we see two highlights)
- Active category will be needed for up/down arrows. We can not just do something activeIndex = += 10 to switch to the next row as the last row in the category may havel less then 10 emojis.